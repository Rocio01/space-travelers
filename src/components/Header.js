import React from 'react';
import { NavLink } from 'react-router-dom';

import planet from '../assets/planet.png';

const Header = () => (
  <div className="panel-bg container">
    <nav className="nav row">
      <div className="col-8 row">
        <img src={planet} alt="planet-logo" className="planet-logo col-4" />
        <h1 className="title  col-8">Space Travelers Hub</h1>
      </div>
      <div className="col-4">
        <NavLink to="/" activeClassName="active-link">
          <span className="rockets-nav mx-3">Rockets</span>
        </NavLink>
        <NavLink to="/missions" activeClassName="active-link">
          <span className="missions-nav mx-3">Missions</span>
        </NavLink>
        <NavLink to="/Profile" activeClassName="active-link">
          <span className="profile-nav mx-3">Profile</span>
        </NavLink>
      </div>
    </nav>
  </div>
);

export default Header;
