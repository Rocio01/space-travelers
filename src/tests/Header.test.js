import React from 'react';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Header';

test('renders to Nav', () => {
  render(
    <BrowserRouter>
      <Header />
    </BrowserRouter>,
  );
  const navText = screen.getByText('Space Travelers Hub');
  expect(navText).toMatchSnapshot();
});
