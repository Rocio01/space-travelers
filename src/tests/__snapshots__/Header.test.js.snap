// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to Nav 1`] = `
<h1
  class="title  col-8"
>
  Space Travelers Hub
</h1>
`;
